markdown/org/docs/designs/shelly/instructions/en.md

```diff
@@ -0,0 +1,92 @@
+	 - Sleeve ease: -20% to -10%. Compression here can be beneficial, but you don't want it so tight that you cut off circulation.
+	 - Wrist ease: -15% to 0%. Mostly preference.
+     - Neck ease: 25% to 100% (25% will still be a very snug neckband that will take some stretching to get on/off the head, because of how the neckband is constructed and because neckholes usually are quite a bit bigger than the neck to fit over the head.
+
+   - Athletic shirt
+     - The author hasn't made one yet. If someone knows what eases to use, please let me know.
+	 - Highly breathable fabric with some spandex/stretch is recommended.
+	 - Eases are somewhere between those for a compression shirt and a swim shirt.
+
+   - Swim shirt (looser fitting stretch shirt meant for swimming)
+     - Swim fabric (spandex/nylon blend or spandex/polyester blend, around 20% spandex)
+     - Chest ease: ~0%
+	 - Sleeve ease: 0-15%
+	 - Neck ease: 50 to 150% (mostly a style decision). Ease under 50% isn't recommended unless you like very snug neckbands.
+
+   - T-shirt
+     - The author hasn't made one yet. If someone knows what eases to use, please let me know.
+	 - Eases will be quite a bit more larger (more positive) than for other shirt types.
+
+   - Sweatshirt
+     - The author hasn't made one yet. If someone knows what eases to use, please let me know.
+	 - Eases will be similar to that for a t-shirt, or a little larger.
+
+ - The neckband length will be mostly determined by the material of the neckband. Stretch fabric will generally use a length of around 80%. Less stretchy fabric will be longer, but always shorter than 100%, and ribbed knit (traditional t-shirt neckbands) are around 60-70%.
+
+### Step 1: Forming the body and attaching the sleeves
+
+- With _good sides together_, sew the front piece to each sleeve along the raglan seam (the diagonal seam running from the neck to the armpit).
+- With _good sides together_, sew the back piece to each sleeve along the raglan seam. You should now have a single piece with a circular neck hole in the middle.
+
+### Step 2: Attaching the neckband
+
+- With _good sides together_, fold the neckband in half long ways (so it's half as long) and sew it into a loop.
+- With _wrong sides together_, fold the neckband in half short ways (so it's half as wide).
+- Turn the shirt right-side-out
+- Mark the quarter points on your neckband with ball point pins, clips, or chalk.
+- Do the same around the neck hole, marking the center of the front, the center of the back, and the midpoints on either sleeve.
+- With _good sides together_, line up the unfinished edges of the neckband with the edge of the neck hole, and line up the quarter points of the neckband with the quarter points of the neck hole. Pin or clip the neckband around the neck hole.
+ - Sew carefully around the neck hole, making sure to sew through all 3 layers.
```

Comment:
Does it need to be stretched to fit?

Reply:
Yeah, it's best to be explicit about it.